pkg/proxy/iptables/proxier.go

```diff
@@ -70,6 +71,8 @@ const kubeMarkMasqChain utiliptables.Chain = "KUBE-MARK-MASQ"
 // TODO(thockin): Remove this for v1.3 or v1.4.
 const oldIptablesMasqueradeMark = "0x4d415351"
 
+const noConnectionToDelete = "0 flow entries have been deleted"
```

Comment:
nit: put this nearer to where it is used?  or wrap it in a util function near where it is used